### PR TITLE
add html , CSS and linters files.

### PR DESCRIPTION
**Pull request by sameh080081**
this pull request for adding html , css , linters and .gitignore

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1 @@
+h1 { color: blue; }
```
Comment (thecodechaser) [inline style.css]: - [x] You are using an element selector to style the heading, Please use a class selector instead to follow the project requirements. You can add to class to the heading and use that class here to style the heading.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @sameh080081👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,58 @@
...
+
+> This is my first project in microverse.
+
+## Built With
+
+- HTML,CSS
+- Github
+- Linters
+
+[Live Demo Link](https://livedemo.com)
```
Comment (topeogunleye) [inline README.md]: - [x] [OPTIONAL] As the previous reviewer did not mention this, I kindly suggest you add a live demo link or remove this section entirely.

**Review by topeogunleye — APPROVED** [review]
### Hi @sameh080081, 

# Status: Approved ✔️
Excellent job on this making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
> 
> ## [Optional] suggestions:
> * [ ]  Please check my inline comments.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**